sys/crypto/aes.c

```diff
@@ -49,7 +71,9 @@ static const cipher_interface_t aes_interface = {
     aes_encrypt,
     aes_decrypt
 };
+
 const cipher_id_t CIPHER_AES_128 = &aes_interface;
+const cipher_id_t CIPHER_AES = &aes_interface;
```

Comment:
Doing a `git grep CIPHER_AES_128`  shows that there are still many uses of the deprecated `cipher_id`, including the crypto documentation. Those should be updated to use `CIPHER_AES` instead.

Reply:
Fixed !